.github/workflows/ci.yml

```diff
@@ -123,18 +123,19 @@ jobs:
       with:
         node-version: 10.x
 
+    - name: Define ToT chrome path
+      if: matrix.chrome-channel == 'ToT'
+      run: echo "::set-env name=CHROME_PATH::/home/runner/chrome-linux-tot/chrome"
```

Comment:
I think set-env only applies to the next step.

Reply:
yeah the wording is stupid https://help.github.com/en/actions/reference/workflow-commands-for-github-actions#setting-an-environment-variable

> Creates or updates an environment variable for any actions **running next** in a job. 

"Next", you say...

> … The action that creates or updates the environment variable does not have access to the new value, **but all subsequent actions in a job will have access**. 


so yeah. it's all subsequent.

also i hate how this doc says "actions" but should say "steps".

----------


tested it earlier: https://github.com/GoogleChrome/lighthouse/runs/805258701?check_suite_focus=true

I read it two steps after i set it.

![image](https://user-images.githubusercontent.com/39191/85636115-7672a180-b634-11ea-9d79-d3240ea52339.png)

Reply:
ah good catch.

cc @github lol